ci/ci_integration.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+rake test:install-all
```

Comment:
Yes I think I will just change it to the ones we bundle logstash with. makes more sense to me.